docs/overview.md

```diff
@@ -1,19 +1,19 @@
 # Kubernetes User Documentation
 
-The Kubernetes API currently manages 3 main resources:
-* [pods](pods.md)
-* [replication controllers](replication-controller.md)
-* [services](services.md)
+Kubernetes is an open source orchestration system for containers (currently Docker containers). It schedules containers onto nodes in a compute cluster and actively manages these containers to ensure that the state of the cluster continually matches the user's intentions.
 
-In Kubernetes, rather than individual containers, _pods_ are the smallest deployable units that can be created, scheduled, and managed. Singleton pods can be created directly, and sets of pods may created, maintained, and scaled using replication controllers.  Services create load-balanced targets for sets of pods.
+In Kubernetes, containers run inside [pods](pods.md), which are an abstraction used to group one or more cooperating containers. Users can create a singleton pod, but the usual deployment pattern in Kubernetes is to define a pod and then to ask the system to create multiple identical replicas of that pod. For each pod the system finds a machine that is healthy and has sufficiently availability capacity, and starts up the corresponding container(s) there.
```

Comment:
Kubernetes has 2 grouping mechanisms: pods and label selectors. I try to avoid the word "group" because it's ambiguous. 

We should say that pods may be comprised of one or more containers and volumes. I think the most common usage will be a container with one or more volumes.